docker/Dockerfile

```diff
@@ -12,8 +12,8 @@
 # limitations under the License.
 
 # syntax=docker/dockerfile:1
-
-FROM registry.access.redhat.com/ubi9/ubi-minimal:latest AS jdk-download
+ARG BASE_IMAGE
```

Comment:
Please mention the motivation in PR description.

Reply:
Done